src/main/java/org/plumelib/util/CombinationIterator.java

```diff
@@ -83,6 +83,7 @@ public boolean hasNext(@GuardSatisfied CombinationIterator<T> this) {
   @RequiresNonNull("nextValue")
   private void advanceNext(@GuardSatisfied CombinationIterator<T> this) {
     for (int i = combinationSize - 1; i >= 0; i--) {
+      assert nextValue != null;
```

Comment:
You need to use a special comment for the Nullness Checker to pick up asserts, see e.g.  https://github.com/eisop/checker-framework/blob/d08def71f8c369a3d85018a5fe4dbd97ed902bd5/dataflow/src/main/java/org/checkerframework/dataflow/analysis/AbstractAnalysis.java#L347

Reply:
Thanks!